### DISCUSSION TITLE
Becoming a Proficient Machine Learning Engineer

### DISCUSSION THREAD

**Discussion by AhmedAlziadi** [mrdbourke/pytorch-deep-learning#498] [General]
I'm thrilled to have found your course, which has been incredibly helpful. However, I've been struggling with writing my own code. Despite being able to copy and run code from your course and other sources successfully, I want to develop the ability to create my own code from scratch. My Python skills are currently at a basic level, and I'm eager to receive your advice on how to progress and become a skilled machine learning engineer."

**Comment by mrdbourke** [comment]
Hi @AhmedAlziadi , 

I've been coding for 6 or so years now and my only advice is to practice, practice, practice.

Set some time each day to practice coding on your own (you can Google etc) but best to just keep writing as much code as possible so you can figure out what works/doesn't work.

Also, see this video: https://www.youtube.com/watch?v=lal7kgZH-0k (learn slower) 

And this article: https://www.mrdbourke.com/learn-slower/
